Makefile

```diff
@@ -102,7 +102,9 @@ generate:
 .PHONY: generate-swagger
 generate-swagger:
 	@hash swagger > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
+		cd .. ; \
```

Comment:
According to `go help modules`
```
If invoked with -mod=readonly, the go command is disallowed from the implicit
automatic updating of go.mod described above. Instead, it fails when any changes
to go.mod are needed. This setting is most useful to check that go.mod does
not need updates, such as in a continuous integration and testing system.
The "go get" command remains permitted to update go.mod even with -mod=readonly,
and the "go mod" commands do not take the -mod flag (or any other build flags).
```
I wonder if `-mod=readonly` could be helpful?

Reply:
it will make go get fail but we just need that it would not update go.mod

Reply:
Yeah, you're right. 🤦‍♂